tests/unit_tests/core/router/transports/ssu/packet.cc

```diff
@@ -36,14 +36,38 @@
 #include <memory>
 
 #include "core/router/transports/ssu/packet.h"
+#include "tests/unit_tests/core/router/identity.h"
+
+namespace core = kovri::core;
 
 /**
  *
  * Global fixtures
  *
  */
 
-struct SSUTestVectorsFixture {
+struct SSUTestVectorsFixture : public IdentityExFixture
+{
+  SSUTestVectorsFixture()
+  {
+    // Build session_confirmed
+    core::OutputByteStream output(session_confirmed.data(), session_confirmed.size());
+    // 1 byte info
+    output.WriteUInt8(0x01);
+    // 2 byte identity size (0x01, 0x87)
+    output.WriteUInt16(m_AliceIdentity.size());
+    // Append identity
+    output.WriteData(m_AliceIdentity.data(), m_AliceIdentity.size());
+    // Signed on time (0x57, 0x69, 0x04, 0xAA)
+    output.WriteUInt32(1466500266);
```

Comment:
>1466500266

Could you please create a definition since this is used throughout several areas?